docs/config/README.md

```diff
@@ -216,7 +216,7 @@ running `eval $(starship init $0)`, and then proceed as normal.
 
 | Variable   | Default         | Description                         |
 | ---------- | --------------- | ----------------------------------- |
-| `min_time` | `2`             | Shortest duration to show time for. |
+| `min_time` | `2000`          | Shortest duration to show time for. |
```

Comment:
Just in case someone skips the intro paragraph, can we clarify that this is in milliseconds?

Reply:
I have added some clarification in the docs now.